pkg/cri/store/image/image.go

```diff
@@ -206,7 +206,7 @@ func (s *store) add(img Image) error {
 		return nil
 	}
 	// Or else, merge and sort the references.
-	i.References = docker.Sort(util.MergeStringSlices(i.References, img.References))
+	i.References = docker.Sort(util.MergeSortedStringSlices(i.References, img.References))
```

Comment:
> Got it, we shouldn't cancel docker.Sort here
PS:Don't worry about the problem of secondary sorting, sorting already ordered sequences is not slow.

Which has better performance than not sorting strings?
I mean why do we need to sort ahead

Reply:
The reason for the early sorting here is that I'm using a List, which is automatically sorted, or perhaps an UnsortedList

Reply:
https://github.com/containerd/containerd/pull/8864 is deprecating `Sort` in favour of `StrictSort`